src/assets/css/styles.css

```diff
@@ -42,7 +42,8 @@
 * - 400 - screen-freeze
 */
 html {
-    overflow: auto !important;
+    overflow-y: scroll !important;
+    overflow-x: auto !important;
```

Comment:
IMHO we shall add a comment on why we have those lines there + link to the issue.
Else we risk that next dev to take them out again, find out why they were there and put them back again :)